package.json

```
@@ -11,18 +11,26 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "prettier": "prettier --write .",
-    "prepare": "husky"
+    "prepare": "husky",
+    "typecheck:bdd": "tsc -p tsconfig.cucumber.json --noEmit",
+    "bdd:dev": "cross-env NODE_OPTIONS='--import tsx/esm' cucumber-js --no-strict -p automated --import 'features/step_definitions/**/*.ts' --import 'features/support/**/*.ts'",
+    "bdd": "tsc -p tsconfig.cucumber.json && cucumber-js -p automated --import 'dist-cucumber/features/**/*.js'" 
+
```

```suggestion
    "bdd:dev": "cross-env NODE_OPTIONS=\"--import tsx/esm\" cucumber-js --no-strict -p automated --import \"features/step_definitions/**/*.ts\" --import \"features/support/**/*.ts\"",
    "bdd": "tsc -p tsconfig.cucumber.json && cucumber-js -p automated --import \"dist-cucumber/features/**/*.js\""
```
Windows requires the use of double quotes. Otherwise the script commands worked with powershell.